lib-object-oriented/line.js

```diff
@@ -26,6 +40,13 @@ export default class Line {
     }
   }
 
+  createTag = variety =>
+    new Tag({
+      tagType: this.tagType,
+      variety: variety,
+      symbolCount: this.symbolCount
+    }).deliverHTMLTag()
```

Comment:
This function is creating a new object of type `Tag`, and then is calling a method on that object. The name `createTag` does not represent the implementation